internal/app/handlers/newUserHandler/newUserHandler_test.go

```diff
@@ -25,9 +26,9 @@ func TestHandler(t *testing.T) {
 		"email":      "hello@stevenxu.me",
 	}
 	jsonStr := utils.ToRawJson(req)
-	rr, err := utils.NewTestRequest("POST", RouteURL, jsonStr, Handler)
+	rr, err := testUtils.NewRequest(ctx, "POST", RouteURL, jsonStr, Handler)
 	if err != nil {
-		t.Errorf("%v", err)
+		t.Error(err)
```

Comment:
This is a lot cleaner. Can you also clean up line 71 and line 45 in the same fashion?

Reply:
ahlie